sql/civicrm_queue_item.mysql

```diff
@@ -16,6 +16,9 @@ CREATE TABLE `civicrm_queue_item` (
      `weight` int NOT NULL   ,
      `submit_time` datetime NOT NULL   COMMENT 'date on which this item was submitted to the queue',
      `release_time` datetime    COMMENT 'date on which this job becomes available; null if ASAP',
+     `retry_interval` int NULL COMMENT 'Number of seconds to wait before retrying a failed execution. NULL to disable.',
```

Comment:
I haven't r-run tested but from my reading of this PR these columns are not added in the upgrade script and the run_count column is added in the xml but not here

Reply:
Snap. This line is totally wrong. It's carry-over from the earlier PR (which had different schema). In this PR, `civicrm_queue_item` should only define the tracking-variable (`run_count`). The policies (`retry_interval`, `retry_limit`) moved to the queue-level.

The mistake here is asymptomic - probably because the `MINIMUM_UPGRADABLE_VERSION` is `4.6` - so the table's pre-existence is now a given. The "pre-upgrade"-upgrade code-path is may still be useful for `ALTER TABLE  civicrm_upgrade_item`, but it's not needed for `CREATE TABLE civicrm_upgrade_item`.

Let me try deleting `civicrm_queue_item.mysql`.